packages/core/src/Stores/traders-hub-store.js

```diff
@@ -423,6 +423,12 @@ export default class TradersHubStore extends BaseStore {
             );
             return;
         }
+
+        if (this.CFDs_restricted_countries) {
+            this.available_dxtrade_accounts = [];
```

Comment:
In my opinion this check must be on line 418 as first check in the function, let me know what do you think? Also, IMHO, we have to get proper accounts from BE and don't filter them on FE side, but it's not related to this PR, it's a global issue :)